.github/workflows/build.yml

```diff
@@ -23,6 +23,7 @@ on:
       - 'docs/**'
       - '*.md'
       - LICENSE
+    types: [opened, synchronize, reopened]
```

Comment:
:question: is this needed by sonar? did you restrict some PR changes events?
I have seen this in the sonarcloud action (https://github.com/SonarSource/sonarcloud-github-action) but I think we can drop it

Reply:
I didn't know if we really want to keep it.
We can remove it 😉